meshparty/trimesh_io.py

```diff
@@ -67,20 +74,27 @@ def write_mesh_h5(filename, vertices, faces,
         if mesh_edges is not None:
             f.create_dataset("mesh_edges", data=mesh_edges, compression="gzip")
 
+        if node_mask is not None:
+            f.create_dataset("node_mask", data=node_mask, compression="gzip")
+
 
-def read_mesh(filename):
+def read_mesh(filename, masked_mesh=False):
     """Reads a mesh's vertices, faces and normals from obj or h5 file"""
 
     if filename.endswith(".obj"):
         vertices, faces, normals = read_mesh_obj(filename)
         mesh_edges = None
     elif filename.endswith(".h5"):
-        vertices, faces, normals, mesh_edges = read_mesh_h5(filename)
+        mesh_data = read_mesh_h5(filename, masked_mesh)
```

Comment:
also needs resolving with #21

Reply:
Definitely. I'm going to wait until we've merged #21 and update.